ide-launcher/src/main/kotlin/com/intellij/remoterobot/launcher/utils.kt

```diff
@@ -27,7 +27,7 @@ fun extractTar(tarFile: Path, to: Path): List<Path> {
 
 fun extractZip(zip: Path, to: Path): List<Path> {
     return getCreatedFilesAfter(to) {
-        FileSystems.newFileSystem(zip, null).use { fs ->
+        FileSystems.newFileSystem(zip).use { fs ->
```

Comment:
Compilation error here since we are building against java 11

Reply:
done